tests/SampleApps/php/wordpress-example/create_wordpress_db.sh

```diff
@@ -3,7 +3,15 @@
 set -ex
 
 apt-get update && apt-get install mariadb-server --yes
-service mysql start
+
+# As of mariadb 10.5, which bullseye is using, the service name is now mariadb
+# instead of mysql, but the mysql command is still symlinked.
+# See: https://mariadb.com/kb/en/changes-improvements-in-mariadb-105/#binaries-named-mariadb-mysql-symlinked
```

Comment:
Just want to call this out explicitly, in case we rely on the mysql service anywhere else but in these tests